_data-prepper/pipelines/configuration/processors/mutate-strings.md

```diff
@@ -0,0 +1,229 @@
+      record_type: "event"
+      format: "json"
+  processor:
+    - split_string:
+        entries:
+          - source: "message"
+            delimiter: ","
+  sink:
+    - stdout:
+```
+
+Create the following file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with your file path:
+
+```json
+{"message": "hello,world"}
+```
+When you run Data Prepper with this `pipeline.yaml` file, you should see the following output:
+
+```json
+{"message":["hello","world"]}
+```
+
+### Configuration
+
+* `entries` (required): A list of entries to add to an event
+* `source` (required): The key to be split
+* `delimiter` (optional): The separator character responsible for the split. Cannot be defined at the same time as `delimiter_regex`. At least `delimiter` or `delimiter_regex` must be defined.
+* `delimiter_regex` (optional): A regex string responsible for the split. Cannot be defined at the same time as `delimiter`. At least `delimiter` or `delimiter_regex` must be defined.
+
+## uppercase_string
+
+The `uppercase_string` processor converts a key to uppercase.
+
+<!--- Converts a key or a string? Are only keys and strings being converted?--->
+
+### Basic usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - uppercase_string:
+        with_keys:
+          - "uppercaseField"
+  sink:
+    - stdout:
+```
+
+Create the following file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with the correct file path:
+
+```json
+{"uppercaseField": "hello"}
+```
+When you run Data Prepper with this `pipeline.yaml` file, you should see the following output:
+
+```json
+{"uppercaseField": "HELLO"}
+```
+
+### Configuration
+
+<!--- Are these paramters? How are these items used? Need some intro text here.--->
+
+* `with_keys` (required): A list of keys to convert to uppercase.
+
+## Lowercase string
+
+The `lowercase string` processor converts a string to lowercase.
+
+### Basic usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - lowercase_string:
+        with_keys:
+          - "lowercaseField"
+  sink:
+    - stdout:
+```
+
+Create the following file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with the path of this file.
+
+```json
+{"lowercaseField": "TESTmeSSage"}
+```
+When you run Data Prepper with this `pipeline.yaml` file, you will see the following output:
+
+```json
+{"lowercaseField": "testmessage"}
+```
+
+### Configuration
+
+<!--- Are these paramters? How are these items used? Need some intro text here.--->
+
+* `with_keys` (required): A list of keys to convert to lowercase
```

Comment:
indent or no indent for bullets. Be consistent throughout file. In this case, you only have one parameter. I don't think we need to set it out with a list bullet. The code formatting may be enough. Try it without the bullet and see if it looks natural and reads clearly.